docs/manuals/gui/viselements/date.md_template

```diff
@@ -34,20 +34,20 @@ a date selector that represents it:
         <taipy:date>{dt}</taipy:date>
         ```
 
-## Using only the date
+## Using the full date and time
 
-If you don't need to use the date, you can do so:
+If you need to use the date and time, you can do so by setting `with_time` property to `True`:
 
 !!! example "Page content"
 
     === "Markdown"
 
         ```
-        <|{dt}|date|not with_time|>
+        <|{dt}|date|with_time=True|>
```

Comment:
```suggestion
        <|{dt}|date|with_time|>
```

Reply:
@FabienLelaquais , thank you for the resourceful suggestion. Does this means `with_time` defaults to `True`  ?
![image](https://github.com/Avaiga/taipy-doc/assets/24577149/06df9e70-37b8-40ed-a62d-73c70e3d096b)